MaterialDesignThemes.Wpf/ComboBoxPopup.cs

```diff
@@ -1,14 +1,21 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Controls.Primitives;
+using System.Windows.Data;
 using System.Windows.Media;
 
 namespace MaterialDesignThemes.Wpf
 {
     public class ComboBoxPopup : Popup
     {
+        public const double ContentPresenterExtend = 4;
+        public const double TopBottomMargin = 8;
```

Comment:
@ButchersBoy move these back to xaml? lol, i moved them to source code because i thought that i need them in code, but really i didn't. Can move it back if you want.